routers/web/repo/view.go

```diff
@@ -578,6 +586,15 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 		}
 	}
 
+	if fInfo.st.IsImage() && !fInfo.st.IsSvgImage() {
+		img, _, err := image.DecodeConfig(bytes.NewReader(buf))
+		if err == nil {
+			// There are Image formats go can't decode
+			// Instead of throwing an error in that case, we show the size only when we can decode
+			ctx.Data["ImageSize"] = fmt.Sprintf("%dx%d", img.Width, img.Height)
```

Comment:
What about using two parameters instead?
That will certainly be better for localization.

Reply:
I think `<num>x<num>` is pretty universal, but can't hurt to change the translation entry to `%dx%d`, named `width_x_height`

Reply:
Did this really need localization? Is {Width}x{Height} not the same in every language?

Reply:
From my point of view, we don't need to translate it.

Reply:
Okay, if it's universal then I'm fine with it.
I've simply learned not to make any assumptions when it comes to i18n.